unlock-app/src/components/interface/buttons/Button.js

```diff
@@ -86,7 +86,7 @@ export const ButtonLink = styled.a`
 export const Label = styled.small`
   display: none;
   position: relative;
-  z-index: 1000;
+  z-index: 30000;
```

Comment:
I think we need to use a couple variables for this because this is going to get complex very quickly.

Reply:
seems maybe a separate PR? Probably best to open an issue for this